src/m365/spo/commands/site/site-classic-set.spec.ts

```diff
@@ -107,6 +107,33 @@ describe(commands.SITE_CLASSIC_SET, () => {
     });
   });
 
+  it('updates site description.', (done) => {
+    sinon.stub(request, 'post').callsFake((opts) => {
+      if ((opts.url as string).indexOf(`https://contoso.sharepoint.com/sites/team/_api/web`) > -1) {
+        if (opts.headers &&
+          opts.headers['X-RequestDigest'] &&
+          opts.headers['X-RequestDigest'] === 'abc' &&
+          opts.headers['X-HTTP-Method'] &&
+          opts.headers['X-HTTP-Method'] === 'MERGE' &&
+          opts.data === "{ '__metadata': { 'type': 'SP.Web' }, 'Description': 'New description' }") {
+          return Promise.resolve();
+        }
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    command.action(logger, { options: { debug: true, url: 'https://contoso.sharepoint.com/sites/team', description: 'New description' } }, () => {
+      try {
+        assert(loggerLogToStderrSpy.called);
```

Comment:
It's more reliable to test for:

```ts
assert.strictEqual(typeof err, 'undefined');
```

`assert(loggerLogToStderrSpy.called);` would be true even if executing the command failed because it would contain the error.